app/components/CustomCard/index.js

```diff
@@ -0,0 +1,37 @@
+/**
+ *
+ * CustomCard
+ *
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import T from '@components/T';
+import { Card } from 'antd';
+import styled from 'styled-components';
+
+const StyledCard = styled(Card)`
+  && {
+    margin: 20px 0;
+    max-width: ${(props) => props.maxWidth} ? ${(props) => props.maxwidth}px : 900px;
```

Comment:
nothing wrong with that but we can also do something like this 
max-width: ${(props) => (props.maxWidth ?? 900)}px;